docs/classified-provisioning-docs/prereqs.md

```diff
@@ -0,0 +1,105 @@
+# Prerequisites
+
+In order to utilize Classified Provisioning in SC2S/C2S, you will need the following prerequisites.
+
+## Tools
+
+The following tools are required to package Classified Provisioning artifacts and install/upgrade Rancher.
+
+- [Hauler](https://rancherfederal.github.io/hauler-docs/docs/introduction/install)
+- [Helm](https://helm.sh/docs/intro/install)
+- [Rancher](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade) (v2.7.13, v2.7.14, v2.8.4 or greater)
+
+## IAM Role & Instance Profile
+
+The following IAM role needs to be created in the account and accessible to attach to the EC2 instances running Rancher MCM, substituting the AWS region and AWS account ID for the placeholders:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "RancherAWSEC2Permissions",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:AuthorizeSecurityGroupIngress",
+                "ec2:Describe*",
+                "ec2:ImportKeyPair",
+                "ec2:CreateKeyPair",
+                "ec2:CreateSecurityGroup",
+                "ec2:CreateTags",
+                "ec2:DeleteKeyPair",
+                "ec2:ModifyInstanceMetadataOptions"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "RancherAWSKMSPermissions",
+            "Effect": "Allow",
+            "Action": [
+                "kms:Decrypt",
+                "kms:GenerateDataKeyWithoutPlaintext",
+                "kms:Encrypt",
+                "kms:DescribeKey",
+                "kms:CreateGrant",
+                "ec2:DetachVolume",
+                "ec2:AttachVolume",
+                "ec2:DeleteSnapshot",
+                "ec2:DeleteTags",
+                "ec2:CreateTags",
+                "ec2:CreateVolume",
+                "ec2:DeleteVolume",
+                "ec2:CreateSnapshot"
+            ],
+            "Resource": [
+              "arn:aws:ec2:AWS_REGION_PLACEHOLDER:AWS_ACCOUNT_ID_PLACEHOLDER:volume/*",
+              "arn:aws:ec2:AWS_REGION_PLACEHOLDER:AWS_ACCOUNT_ID_PLACEHOLDER:instance/*",
+              "arn:aws:ec2:AWS_REGION_PLACEHOLDER':AWS_ACCOUNT_ID_PLACEHOLDER:snapshot/*",
+              "arn:aws:kms:AWS_REGION_PLACEHOLDER':AWS_ACCOUNT_ID_PLACEHOLDER:key/*"
```

Comment:
If we choose to use envvar style placeholders then savvy operators can pipe this through envsubst with the appropriate environment. And if we're doing that, worthwhile to use standard envvars e.g. `${AWS_REGION:-${AWS_DEFAULT_REGION}}`.

Reply:
My only modest feedback to this is it breaks JSON spec. So this either needs to be a pure bash script that calls the AWS CLI (and update the markdown code block annotation) or left as is.

Reply:
good with me... we can worry about it next time...